cluster/manifests/skipper/deployment.yaml

```diff
@@ -3,16 +3,25 @@
 {{ $main_image := print "container-registry.zalando.net/teapot/skipper-internal:" "v0.21.211-1033" }}
 {{ $canary_image := "container-registry.zalando.net/teapot/skipper-internal:v0.21.212-1034" }}
 
-
 {{/* Optional canary arguments separated by "[cf724afc]" to allow whitespaces, e.g. "-foo=has a whitespace[cf724afc]-baz=qux" */}}
 {{ $canary_args := "" }}
 
+{{ $cluster_internal_cidrs := stringSlice "10.2.0.0/15" .Values.vpc_ipv4_cidr }}
+{{ if eq .Cluster.Provider "zalando-eks" }}
+{{ if eq .Cluster.ConfigItems.eks_ip_family "ipv6" }}
+{{ $cluster_internal_cidrs := .Values.vpc_ipv6_cidrs }}
```

Comment:
This should use assignment as `cluster_internal_cidrs` already exists.
Should it include `10.2.0.0/15` or we just uses what's in the value?

Reply:
`10.2.0.0/15` is only relevant for non EKS clusters.